tasks/queue-granules/src/types.ts

```diff
@@ -0,0 +1,93 @@
+/**
+ * This file is generated using @cumulus/schema. Any modifications made to this file
+ * will the overwritten when the build script is rerun. Please do not modify this file.
+ */
+
+/**
+ * Describes the config used by the queue-granules task
+ */
+export interface QueueGranulesConfig {
+  granuleIngestWorkflow: string;
+  internalBucket: string;
+  provider: {
+    id: string;
+  };
+  preferredQueueBatchSize?: number | null;
+  queueUrl: string;
+  stackName: string;
+  concurrency?: number;
+  executionNamePrefix?: string;
+  childWorkflowMeta?: {};
```

Comment:
what's the difference between `childWorkflowMeta?: {}` and `childWorkflowMeta?: object`?

Reply:
https://stackoverflow.com/questions/49464634/difference-between-object-and-object-in-typescript

Seems like `object` is probably best though not that different from `{}`, but `object` is probably a better fit. I don't think I can make that change without writing my own version of json-schema-to-typescript